src/jbuild_load.ml

```diff
@@ -106,6 +118,29 @@ module Jbuild_plugin = struct
       let oc = open_out_bin %S in
       output_string oc s;
       close_out oc
+
+    let run_and_read_lines cmd =
+      let tmp_fname = Filename.temp_file "dune" ".output" in
+      at_exit (fun () -> Sys.remove tmp_fname);
+      let n =
+        Printf.ksprintf Sys.command "%%s > %%s" cmd (Filename.quote tmp_fname)
+      in
+      let rec loop ic acc =
+        match input_line ic with
+        | exception End_of_file -> close_in ic; List.rev acc
+        | line -> loop ic (line :: acc)
```

Comment:
Can we move this to the `Io` module? Note that I see so much opportunity for code reuse, but keeping the application code uncluttered is nice.

Reply:
It's not a trivial move since this is for plugins

Reply:
Ah yeah, that's right. Bit of a shame.